src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1417,6 +1485,51 @@ export const handleGetSettings: ControllerHandler<
     })
 }
 
+/**
+ * Handler for GET api/platform/v1/admin/forms/:formId/webhookSettings.
```

Comment:
Is there an existing endpoint for non-platform calls like `GET api/v1/admin/forms/:formId/webhookSettings.` ? Given that whether a user is an admin or not is abstracted from the caller, I wonder if it needs to be in the path. 

In future, if we want to expose an originally platform api to all users, it could be done by modifying the logic internally, instead of creating a separate endpoint for non platform users.

Reply:
> In future, if we want to expose an originally platform api to all users, it could be done by modifying the logic internally, instead of creating a separate endpoint for non platform users.

Yeah great point! After the discussion with the team last Friday, I think it makes sense to have just one endpoint. Will be modifying this PR accordingly and request for a review again after

Reply:
> Is there an existing endpoint for non-platform calls like GET api/v1/admin/forms/:formId/webhookSettings. ? Given that whether a user is an admin or not is abstracted from the caller, I wonder if it needs to be in the path.

There isn't, it's just `/admin/forms/:formId/settings`. After thinking about it further + discussing with @LinHuiqing and @foochifa , I think we can just use that (not restricting the set of settings returned and updated), since we're using the same endpoint for platform and non-platform users. 

The initial consideration (about how limiting the endpoint makes this more secure) is moot because
- platform and non-platform users have the same access
- if they're API users, we can assume that they're fairly tech savvy, the likelihood that they mistakenly change the wrong setting is low.

I will filter out some of the settings fields that we don't want any admin to be able to edit, like `payments_channel` and `publicKey`.